client/src/routes/screens/Home/Home.tsx

```diff
@@ -217,6 +224,23 @@ const Home = () => {
     navigate('AboutOverlay');
   }, [navigate]);
 
+  const throttledRefresh = useMemo(
+    () => throttle(() => refreshPull(), 5 * 60000),
+    [refreshPull],
+  );
+
+  useEffect(() => {
+    if (isFocused) {
+      throttledRefresh();
+    }
+  }, [isFocused, throttledRefresh]);
+
+  useResumeFromBackgrounded(() => {
+    if (isFocused) {
+      throttledRefresh();
+    }
```

Comment:
Since you do the same check in these two hooks, maybe you could move it inside of the throttled function?

Reply:
done [6479f65](https://github.com/29ki/29k/pull/2804/commits/6479f6536378430f562f65a514d08eac4ecd623c)